### ISSUE TITLE
packaging for pip or poetry

### ISSUE THREAD

**Issue by PeterNSteinmetz** [jniediek/combinato#78]
It appears there is no setup.py or pyproject.toml file. Addition of these would allow people to install combinato as a package from source and might be handy.

**Comment by jniediek** [comment]
Yes, you are absolutely right. Combinato should have been available in pypi for a long time, I just never found the time to add it. Would you be willing to create a pull request?
Cheers
Johannes

**Comment by PeterNSteinmetz** [comment]
Unfortunately I don't have a lot of experience with python packaging. Perhaps one of the other users might pitch in on this?
